jaeger_client/thrift.py

```diff
@@ -13,37 +13,48 @@
 # limitations under the License.
 
 import traceback
-from opentracing.tracer import ReferenceType
-from .constants import MAX_TRACEBACK_LENGTH
+from typing import Any, List, Tuple, Optional, TYPE_CHECKING
+from types import TracebackType
+
+from opentracing.tracer import ReferenceType, Reference
 
+from .constants import MAX_TRACEBACK_LENGTH
 import jaeger_client.thrift_gen.jaeger.ttypes as ttypes
 import jaeger_client.thrift_gen.sampling.SamplingManager as sampling_manager
 
+if TYPE_CHECKING:
```

Comment:
fyi, this syntax is needed to avoid circular imports. If a type from this section is used it must be wrapped in quotes. Mypy will the dynamically resolve the type